### PR TITLE
fix setting tuned profile

### PR DESCRIPTION
**Pull request by dangel101**
move profile setting from ansible role to engine

### CODE REVIEW

File: backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/hostdeploy/InstallVdsInternalCommand.java [hunk 1]
```diff
@@ -277,7 +277,9 @@ private void runAnsibleHostDeployPlaybook() {
             }
             Cluster hostCluster = clusterDao.get(getClusterId());
             boolean isGlusterServiceSupported = hostCluster.supportsGlusterService();
-            String tunedProfile = isGlusterServiceSupported ? hostCluster.getGlusterTunedProfile() : null;
+            boolean isVirtEnabled = hostCluster.supportsVirtService();
+            String tunedProfile = isVirtEnabled && !isGlusterServiceSupported ? "virtual-host"
+                    : isGlusterServiceSupported ? hostCluster.getGlusterTunedProfile() : null;
```
Comment (mwperina) [inline backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/hostdeploy/InstallVdsInternalCommand.java:282]: Does this fix the issue that gluster services are enabled for the cluster, but gluster tuned profile is set to NULL due to the UI bug? Wouldn't below condition be more clear?

```
String tunedProfile = "virtual-host";
if (hostCluster.supportsGlusterService() && StringUtils.isNotBlank(hostCluster.getGlusterTunedProfile())) {
    // different tuned profile is specified for gluster cluster
    tunedProfile = hostCluster.getGlusterTunedProfile();
}
```

**Review by mwperina — CHANGES_REQUESTED** [review]

File: backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/hostdeploy/InstallVdsInternalCommand.java (hunk 1 above)
Comment (dangel101) [reply backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/hostdeploy/InstallVdsInternalCommand.java]: yes, done

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**